arch/arm/boot/dts/wireless-murata-sd8987.dtsi

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020 CompeGPS Team S.L.
+ * All rights reserved
+ *
+ */
+
+/ {
+	init-gpios {
+		sd8987_switch {
+			label = "sd8987_switch";
+			gpios = <&gpio1 31 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+		sd8987_enable {
+			label = "sd8987_enable";
+			gpios = <&gpio1 29 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+	};
+};
+
+/* Bluetooth UART */
+&uart2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart2>;
+	fsl,uart-has-rtscts;
+	status = "okay";
+};
+
+&usdhc1 {
+	pinctrl-names = "default", "state_100mhz", "state_200mhz";
+	pinctrl-0 = <&pinctrl_usdhc1>;
+	pinctrl-1 = <&pinctrl_usdhc1_100mhz>;
+	pinctrl-2 = <&pinctrl_usdhc1_200mhz>;
+	fsl,tuning-start-tap = <20>; /* test this */
+	bus-width = <4>;
+	broken-cd;
+	enable-sdio-wakeup;
+	keep-power-in-suspend;
+	status = "okay";
+};
+
+&iomuxc {
+	imx6ul-evk {
+		pinctrl_wlan_ctrl: wlan-ctrl {
```

Comment:
As I told to you today, I think pinctrl groups must be referenced in order to be initialized (i think this is in progress)